scripts/pychop/merlin.yaml

```diff
@@ -21,7 +21,7 @@ chopper_system:
       radius: 300               # Disk radius
       isDouble: False           # Is this a double disk system?
       isPhaseIndependent: True  # Is this disk to be phased independently?
-      defaultPhase: 12800       # What is the default phase for this disk (either a time in microseconds
+      defaultPhase: 12400       # What is the default phase for this disk (either a time in microseconds
```

Comment:
The original issue mentions the disk phase
> should be kept fixed at 12800us

Just wanted to check that this change is expected.

Reply:
Yes, sorry the `12800` was a mistake on my part. I originally wrote this down but when I checked with the actual value used on the instrument it was `12400` but I didn't change the text in the original issue. Silke messaged me on Slack about it and we agreed to use `12800` but didn't document it on github.